### PR TITLE
Add toggle for interactive JSON viewer.

### PR DESCRIPTION
**Pull request by AcylSilane**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,4 +1,4 @@
-jupyterlab==3.0.3
+jupyterlab==2.2.7
```
Comment (timurbazhirov) [inline requirements.txt:1]: Let's keep 3.0.3 pls

File: examples/job/run-simulations-and-extract-properties.py [hunk 2]
```diff
@@ -96,7 +96,7 @@
 
 # #### Workflow
 # 
-# This example is based on [this](https://platform.exabyte.io/analytics/workflows/56xDyXsPMNJ7cF9nv) bank workflow which is later copied to the account workflows collection. The workflow is named "D3-GGA-BS-BG-DOS-ALL" and utilizes the logic explained in https://arxiv.org/pdf/1808.05325.pdf, for example (see section "Methodology", Table I). "D3" indicates the difficulty level 3 per the table convention. BS, BG, DOS indicate the properties extracted - Band Structure, Band Gap, Density of States. The workflow is utilizing VASP simulation engine at version 5.4.4.
+# This example is based on [this](https://platform.exabyte.io/analytics/workflows/56xDyXsPMNJ7cF9nv) bank workflow which is later copied to the account workflows collection.
```
Comment (timurbazhirov) [inline examples/job/run-simulations-and-extract-properties.py:99]: Let's keep the message pls

File: requirements.txt (hunk 1 above)
Comment (AcylSilane) [reply requirements.txt]: Version bumped back up to 3.0.3

File: examples/job/run-simulations-and-extract-properties.py (hunk 2 above)
Comment (AcylSilane) [reply examples/job/run-simulations-and-extract-properties.py]: Added the message back in

**Review by timurbazhirov — APPROVED** [review]

**Pull request merged**